edward/util.py

```diff
@@ -357,17 +471,25 @@ def to_simplex(x):
     Parameters
     ----------
     x : tf.tensor or np.array
-        scalar, vector, or matrix
+        vector, or matrix
```

Comment:
this should work for scalar as well (i.e., all tf.tensors). we can do a sweep in another pull request changing how we write documentation of tensors; see #191